Demo/Common/IntegrationExample.swift

```diff
@@ -29,9 +30,12 @@ internal final class IntegrationExample: APIClientAware {
 
     internal var paymentMethods: PaymentMethods?
     internal var currentComponent: PresentableComponent?
+    internal var sessionPaymentMethods: PaymentMethods?
```

Comment:
Why can't we use existing `paymentMethods` ? 🤔

Reply:
Every time switch is changed, we need to change the active payment methods between old response and session response